src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -50,6 +55,130 @@ public void parseIndex_validInput_success() throws Exception {
         assertEquals(INDEX_FIRST_PERSON, ParserUtil.parseIndex("  1  "));
     }
 
+
+    //=========== parsePersonSortField() Tests ================================================================
+
+    @Test
+    public void parsePersonSortField_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> ParserUtil.parsePersonSortField((String) null));
+    }
+
+    @Test
+    public void parsePersonSortField_invalidValue_throwsParseException() {
+        // Invalid letter
+        assertThrows(ParseException.class, () -> ParserUtil.parsePersonSortField("x"));
+
+        // Multiple letters
+        assertThrows(ParseException.class, () -> ParserUtil.parsePersonSortField("xyz"));
+
+        // Whitespace
+        assertThrows(ParseException.class, () -> ParserUtil.parsePersonSortField(WHITESPACE));
+    }
+
+    @Test
+    public void parsePersonSortField_validValueWithoutWhitespace_returnsSortField() throws Exception {
```

Comment:
The purpose of this test case is to use the `assertEquals()` method, not to throw an exception. However, `ParserUtil.parsePersonField()` throws a `ParseException` and Java requires that I handle potential exceptions. Hence, this test method will throw exceptions since it doesn't handle them using a `try` block.